### ISSUE TITLE
identity mapping loss 

### ISSUE THREAD

**Issue by iariav** [vanhuyz/CycleGAN-TensorFlow#24]
Hi,
thank you for sharing this implementation.
do you plan on adding support for the identity mapping loss as described in the original paper?
thanks

**Comment by vanhuyz** [comment]
Yes, I will add identity mapping loss soon. But before that, I have to solve #18.
Anyway, pull requests are always welcome! 😉

**Comment by lizhuorong** [comment]
Have you added the identity mapping loss yet? Besides, I found the training is slow. Do you have any idea to facilitate the training? Have you think using WGAN-GP instead of LSGAN?

**Comment by lizhuorong** [comment]
Have you added the identity mapping loss yet? Besides, I found the training is slow. Do you have any idea to facilitate the training? What do you think of using WGAN-GP instead of LSGAN?

**Comment by Zxr1314** [comment]
@vanhuyz  Good luck

**Comment by Zxr1314** [comment]
@lizhuorong CycleWGAN should be a more robust method

**Comment by hala3** [comment]
i want to change the loss function by pixel wise loss , how i can do that please
